tests/js/server/aql/aql-optimizer-rule-late-document-materialization.js

```diff
@@ -342,8 +342,8 @@ function lateDocumentMaterializationRuleTestSuite () {
                     "LET e = SUM(FOR c IN " + collectionNames[(i + 1) % numOfCollectionIndexes] + " LET p = CONCAT(c.obj.b, c.obj.a) RETURN p) " +
                     "SORT CONCAT(a, e) LIMIT 10 RETURN d";
         let plan = AQL_EXPLAIN(query).plan;
+        assertNotEqual(-1, plan.rules.indexOf(ruleName));
         if (!isCluster) {
```

Comment:
Should we enable full check for the query result in the cluster case? As the rule is now applied and we have to validate the results